saga-core/src/test/java/io/servicecomb/saga/core/SagaCoordinatorTest.java

```diff
@@ -135,8 +168,20 @@ public void runSagaWithEventStore() {
 
   @Test
   public void processRequestsInParallel() {
-    CompletableFuture.runAsync(() -> coordinator.run(requestJson));
-    CompletableFuture.runAsync(() -> coordinator.run(anotherRequestJson));
+    CompletableFuture.runAsync(() -> {
```

Comment:
coordinator is not supposed to throw checked exception

Reply:
Done